### PR TITLE
Update Oregon Trail

### PR DESCRIPTION
**Pull request by hannahhch**

### CODE REVIEW

File: oregon-trail/test/gear-test.js [hunk 1]
```diff
@@ -21,18 +25,15 @@ describe('Gear', function() {
     var clothes = new Gear('clothes');
     var invalid = new Gear('mp3 player');
 
+    var valid = ammunition.checkForValidity();
+    var invalid = invalid.checkForValidity();
+
+    assert.equal(valid, 'Great, we\'ll need lots of ammunition!');
```
Comment (kaylaewood) [inline oregon-trail/test/gear-test.js:31]: much better!

File: oregon-trail/test/settler-test.js [hunk 2]
```diff
@@ -3,115 +3,115 @@ var Settler = require('../src/settler.js');
...
-    traveler.experienceDistress('dysentery')
+    traveler.experienceDistress('broken arm');
+    traveler.experienceDistress('dysentery');
 
     assert.deepEqual(traveler.ailments, ['broken arm', 'dysentery']);
     assert.equal(traveler.status, 'poor');
-  })
+  });
 
-  it.skip('should be able to be killed', function() {
```
Comment (kaylaewood) [inline oregon-trail/test/settler-test.js:72]: hahah too dark?

File: oregon-trail/test/settler-test.js [hunk 3]
```diff
@@ -3,115 +3,115 @@ var Settler = require('../src/settler.js');
...
-    traveler.heal()
+    traveler.experienceDistress('broken arm');
+    traveler.experienceDistress('snake bite');
+    traveler.experienceDistress('dysentery');
+    traveler.heal();
 
     assert.deepEqual(traveler.ailments, ['broken arm', 'snake bite', 'dysentery']);
     assert.equal(traveler.status, 'dead');
-  })
+    assert.equal(traveler.heal(), 'Sorry, we can\'t heal a corpse. Will needs a proper burial!');
```
Comment (kaylaewood) [inline oregon-trail/test/settler-test.js:115]: love it

File: oregon-trail/test/wagon-test.js [hunk 4]
```diff
@@ -193,110 +174,110 @@ describe('Wagon', function() {
...
   it.skip('should not be able to travel without at least one living settler', function() {
-    var wagon = makeValidWagon()
+    var wagon = makeValidWagon();
 
-    wagon.settlers[0].status = 'dead'
+    wagon.settlers[0].status = 'dead';
 
-    assert.isFalse(wagon.canTravel())
-  })
+    assert.isFalse(wagon.canTravel());
```
Comment (kaylaewood) [inline oregon-trail/test/wagon-test.js:281]: so. many. semicolons.

**Review by kaylaewood — APPROVED** [review]

File: oregon-trail/test/settler-test.js (hunk 2 above)
Comment (hannahhch) [reply oregon-trail/test/settler-test.js:72]: like yesss, but also its oregon trail?! I was unsure

**Pull request merged**